package.json

```diff
@@ -42,6 +42,6 @@
     "smooth-scrollbar": "^5.2.1"
   },
   "peerDependencies": {
-    "react": "^0.14.7"
+    "react": "^0.14.0 || ^15.0.0"
```

Comment:
OMG it's version 15.x!!! I seem to stay in school so long that I've missed a lot things outside 😱